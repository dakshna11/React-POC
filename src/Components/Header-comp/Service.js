import React from 'react';
import car from '../img/car.png'
import window from '../img/window.png'
import vacuum from '../img/vacuum-cleaner.png'
import seat from '../img/seat.png'
import interior from '../img/interior.png'
import oil from '../img/car-oil.png'
import brake from '../img/brake.png'
function Service(props) {
    return (
        <div>
            <div className="about-us">
            <div className='about-container' >
                <h1 className="about-text">Service</h1>
                <p>Home / Service</p>
            </div> 
            </div> 
            <div>
                <p id='service-heading'>WHAT WE DO?</p>
                <h1 style={{textAlign:'center', color:'#202c45'}}>Premium Washing Services</h1>
            </div>
            <div className='content1'>
                <div>
                    <img src={car} height='50px' width='50px'></img>
                    <h3>Exterior Washing</h3>
                    <p>Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor</p>
                </div>
                <div style={{marginLeft:'2%'}}>
                    <img src={interior} height='50px' width='50px'></img>
                    <h3>Interior Washing</h3>
                    <p>Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor</p>
                </div>
                <div style={{marginLeft:'2%'}}>
                    <img src={vacuum} height='50px' width='50px'></img>
                    <h3>Vaccum Cleaning</h3>
                    <p>Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor</p>
                </div>
                <div style={{marginLeft:'2%'}}>
                    <img src={seat} height='50px' width='50px'></img>
                    <h3>Seats Washing</h3>
                    <p>Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor</p>
                </div>
            </div>
            <div className='content2'>
                <div style={{marginLeft:'2%'}}>
                        <img src={window} height='50px' width='50px'></img>
                        <h3>Window Wiping</h3>
                        <p>Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor</p>
                </div>
                <div style={{marginLeft:'2%'}}>
                    <img src={window} height='50px' width='50px'></img>
                    <h3>Wet Cleaning</h3>
                    <p>Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor</p>
                </div>
                <div style={{marginLeft:'2%'}}>
                    <img src={oil} height='50px' width='50px'></img>
                    <h3>Oil Changing</h3>
                    <p>Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor</p>
                </div>
                <div style={{marginLeft:'2%'}}>
                    <img src={brake} height='50px' width='50px'></img>
                    <h3>Brake Repair</h3>
                    <p>Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor</p>
                </div>
            </div>
            
        </div>
    );
}

export default Service;