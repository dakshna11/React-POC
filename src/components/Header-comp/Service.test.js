import React from "react";
import { shallow } from "enzyme";
import Service from "./Service";

const service = 'Test Service'
let wrapped = shallow(<Service>{service}</Service>)

describe('Service', ()=>{
    it('should render the Service Component correctly', ()=>{
        expect(wrapped).toMatchSnapshot()
    })
})