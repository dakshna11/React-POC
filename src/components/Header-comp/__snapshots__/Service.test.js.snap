// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Service should render the Service Component correctly 1`] = `
<div>
  <div
    className="about-us"
  >
    <div
      className="about-container"
    >
      <h1
        className="about-text"
      >
        Service
      </h1>
      <p>
        <Link
          className="wordsWP"
          to="/Service"
        >
          Home 
        </Link>
        /
        <Link
          className="wordsWP"
          to="/Service"
        >
           Service
        </Link>
      </p>
    </div>
  </div>
  <div>
    <p
      id="service-heading"
    >
      WHAT WE DO?
    </p>
    <h1
      style={
        Object {
          "color": "#202c45",
          "textAlign": "center",
        }
      }
    >
      Premium Washing Services
    </h1>
  </div>
  <div
    className="content1"
  >
    <div>
      <img
        height="50px"
        src="car.png"
        width="50px"
      />
      <h3>
        Exterior Washing
      </h3>
      <p>
        Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor
      </p>
    </div>
    <div
      style={
        Object {
          "marginLeft": "2%",
        }
      }
    >
      <img
        height="50px"
        src="interior.png"
        width="50px"
      />
      <h3>
        Interior Washing
      </h3>
      <p>
        Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor
      </p>
    </div>
    <div
      style={
        Object {
          "marginLeft": "2%",
        }
      }
    >
      <img
        height="50px"
        src="vacuum-cleaner.png"
        width="50px"
      />
      <h3>
        Vaccum Cleaning
      </h3>
      <p>
        Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor
      </p>
    </div>
    <div
      style={
        Object {
          "marginLeft": "2%",
        }
      }
    >
      <img
        height="50px"
        src="seat.png"
        width="50px"
      />
      <h3>
        Seats Washing
      </h3>
      <p>
        Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor
      </p>
    </div>
  </div>
  <div
    className="content2"
  >
    <div
      style={
        Object {
          "marginLeft": "2%",
        }
      }
    >
      <img
        height="50px"
        src="window.png"
        width="50px"
      />
      <h3>
        Window Wiping
      </h3>
      <p>
        Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor
      </p>
    </div>
    <div
      style={
        Object {
          "marginLeft": "2%",
        }
      }
    >
      <img
        height="50px"
        src="window.png"
        width="50px"
      />
      <h3>
        Wet Cleaning
      </h3>
      <p>
        Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor
      </p>
    </div>
    <div
      style={
        Object {
          "marginLeft": "2%",
        }
      }
    >
      <img
        height="50px"
        src="car-oil.png"
        width="50px"
      />
      <h3>
        Oil Changing
      </h3>
      <p>
        Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor
      </p>
    </div>
    <div
      style={
        Object {
          "marginLeft": "2%",
        }
      }
    >
      <img
        height="50px"
        src="brake.png"
        width="50px"
      />
      <h3>
        Brake Repair
      </h3>
      <p>
        Lorem ipsum dolor sit amet elit. Phase nec preti facils ornare velit non metus tortor
      </p>
    </div>
  </div>
</div>
`;
